src/Types.js

```diff
@@ -78,6 +80,14 @@ class Types {
         return this.get('required');
     }
 
+    defaultValue(value: any) {
+        if (value || false) {
```

Comment:
This is going to ignore other falsy values such as `0`. We can probably test that value is `undefined`. Something along the lines of:

```javascript
if (value === undefined) {
  return this.get('defaultValue');
}

return this.set('defaultValue', value);
```

If the `defaultValue` is initialized, we don't need the `|| null` part anymore.

Reply:
right, changing for `value === undefined`.